src/components/popup/LogIn.js

```diff
@@ -1,42 +1,75 @@
 import React from 'react';
-import { Modal, ModalBody, Button } from 'reactstrap';
+import { Modal, ModalBody, Button, Form, FormGroup, Input, Label } from 'reactstrap';
 import { connect } from 'react-redux';
 import { login } from '../../actions/auth';
 
+class loginModal extends React.Component {
 
-const loginModal = ({ auth, hideModals, login }) => {
-  const handleAdminLogin = event => {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      errors: {}
+    };
+
+    this.handleLogin = this.handleLogin.bind(this);
+  };
+
+  handleLogin = event => {
     event.preventDefault();
+
+    const { login } = this.props;
+
+    console.log(this.state.username);
+
     login({
-      username: 'admin',
-      password: 'admin'
+      username: this.state.username,
+      password: this.state.password
     });
+
   };
-  
-  const handleUserLogin = event => {
-    event.preventDefault();
-    login({
-      username: 'user',
-      password: 'user'
+
+  updateUsernameValue = event => {
```

Comment:
lepší je vytvořit jednu metodu, a podle namu z eventy to připisovat do správné proměnné ve statu